deploy/ansible/roles/goshimmer-node/templates/docker-compose-entrynode.yml.j2

```diff
@@ -0,0 +1,24 @@
+#jinja2: trim_blocks:True, lstrip_blocks:True
+version: "3"
+
+services:
+  entrynode:
+    image: iotaledger/goshimmer:latest
+    container_name: entrynode
+    restart: unless-stopped
+    network_mode: "host"
+    volumes:
+      - "./db:/db"
+      - "/etc/localtime:/etc/localtime:ro"
+    environment:
+      - DATABASE_DIRECTORY=/db/mainnetdb
+      - AUTOPEERING_SEED={{ entryNodeSeed }}
+      - PROFILING_BINDADDRESS=0.0.0.0:6060
+    entrypoint: /run/goshimmer
+    command: >
+      --skip-config=true
+      --autopeering.port=15626
```

Comment:
I would add, just to be more sure that we will never ever mix this network with the public DevNet also a different networkVersion (e.g., `--autopeering.networkVersion=8000`).
And the same should also be replicated for all the other GoShimmer nodes

Reply:
Sure